doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -261,19 +327,60 @@ There are four sorts :g:`SProp`, :g:`Prop`, :g:`Set`  and :g:`Type`.
 
 More on sorts can be found in Section :ref:`sorts`.
 
+.. _fix:
+
+Fix and Cofix
```

Comment:
This section actually already exists: it is called "Recursive functions".

Reply:
OK.  I think the section title should be "Recursive functions: fix and cofix" OR "Fix and cofix: recursive functions" to make it easier to find things.  And same idea for the rest of the chapter.  What do you think?  Doesn't have to be done now.

Reply:
Let's go for "Recursive and co-recursive functions: fix and cofix".